app/demo-loop/simple-driver.py

```diff
@@ -63,7 +64,7 @@
     'brem_combined': True,
     'brem_lpm': True,
     'conv_lpm': True,
-    'enable_msc': False,
+    'enable_msc': use_vecgeom,
```

Comment:
If msc is enable as the default, max_steps should be adjusted (to 512).

Reply:
Thanks, will do!